From: Benjamin Kaduk <kaduk@mit.edu>
Date: Thu, 15 Dec 2016 22:29:41 -0600
Subject: softsig-helper: fix signame() loop

We want to iterate through the length of the array, not whatever
sizeof(<length of the array>) happens to evaluate to.

Change-Id: I5d939e241e848ed800524d5f501e0504d2b0b42d
---
 tests/opr/softsig-helper.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/opr/softsig-helper.c b/tests/opr/softsig-helper.c
index 91fa351..416f07c 100644
--- a/tests/opr/softsig-helper.c
+++ b/tests/opr/softsig-helper.c
@@ -55,7 +55,7 @@ static struct sigtable {
 static char *signame(int signo) {
     int i;
 
-    for (i = 0; i < sizeof(sizeof(sigtable) / sizeof(sigtable[0])); ++i) {
+    for (i = 0; i < sizeof(sigtable) / sizeof(sigtable[0]); ++i) {
 	if (sigtable[i].signo == signo) {
 	    return sigtable[i].name;
 	}
